source/taup/taup_slowplot.rst

```diff
@@ -0,0 +1,44 @@
+taup slowplot
+=============
+
+:本节贡献者: |田冬冬|\（作者）、
+             |姚家园|\（审稿）
+:最近更新日期: 2021-06-01
+
+----
+
+``taup slowplot`` 可以很方便地生成用于绘制慢度-深度剖面的 GMT 脚本。
+
+慢度定义
+---------
+
+此处的慢度定义为 :math:`s = \frac{r}{v}`\ ，单位为 s/rad。在球坐标系下，
+射线参数（即水平慢度）的定义为 :math:`p = \frac{r \sin\theta}{v}`\ ，其中
+:math:`\theta` 是射线与径向的夹角。在地震波传播过程中，射线参数保持不变。
+因此，由于地球不同深度处的半径 r 和速度 v 不同，角度 :math:`\theta` 也会发生变化。
+在某一深度处，若 :math:`\theta = 90`\ ，则射线将在该深度折返。
+因而慢度-深度模型展示了某个射线参数对应的射线所能采样的最大深度。
```

Comment:
Just one note: For P waves, we have two 300 s/rad slowness (one if in the lower mantle and the other in the outer core). Actually, we don't have a P wave with a ray parameter of 300 s/rad turning in the outer core, because the ray will turn in the lower mantle (See https://seismo-learn.org/seismology101/seismology/introduction/#id2). So, the slowness in the outer core in this slowness-depth figure might not exist at all.

Reply:
Good point. I think the description is not 100% correct. 

The slowness in the plot is calculated s=r/v. Thus, it definitely exists in the outer core.

Reply:
I find another interesting point. See PKP time curve and ray path:

![PKP curve](https://user-images.githubusercontent.com/50591376/120822398-f44f6480-c588-11eb-81ea-716a493f8944.png)

![ray path](https://user-images.githubusercontent.com/50591376/120822563-1e088b80-c589-11eb-91e7-f2e04a9db0c1.png)

When the P wave touches the CMB, the ray path is Point A, which arrives at the largest distance (176.91 deg for a 10 km depth source)

```bash
$ taup pierce -mod iasp91 -h 10 -ph PKP -deg 176.91
> PKP at  1313.22 seconds at   176.91 degrees for a     10.0 km deep source in the iasp91 model with rayParam    4.439 s/deg.
   2.52   660.0    80.1
   49.18  2889.0   408.0
   88.44  3955.4   655.7
  127.71  2889.0   903.4
```

For rays from Point A to Point B (PKPab), the gcarc decreases, while the ray parameter and turning depth also decreases (theta also decreases).

For rays from Point B to Point C (PKPbc), the gcarc increases, while the ray parameter and turning depth decreases (theta also decreases).

Therefore, the slowness near the CMB will correspond to that of PKPab with the maximum turning depth and the largest distance. I checked the slowness-depth file. 
```
2889.0 254.33138
...
3955.3672 254.33138 
```

It means that PKPab near 179 deg will turn at the depth of ~3955.37 km, which is consistent with the result from `taup pierce`.

Reply:
If my understanding above is right, the slowness-depth figure can tell us the PKPab maximum turning depth quickly. Just plot a vertical line for the slowness at CMB. The crossing point in the outer core corresponds to PKPab maximum turning depth at ~176.9 deg.

Reply:
@seisman

Reply:
> The ray parameters between depths from 2889 and 3955 DO NOT exist for P wave.

That's why it is called shadow zone and difficult to be sampled seismically.